include/RAJA/config.hxx.in

```diff
@@ -154,102 +154,106 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 //
 //     RAJA_INLINE - macro to enforce method inlining
 //
-//     RAJA_ALIGN_DATA(<variable>) - macro to express alignment of data, 
-//                              loop bounds, etc.
+//     RAJA_SIMD - macro to apply compiler-specific pragma that encourages
+//                 compiler to SIMD vectorize loop even if it may not
+//                 normally
+//
+//     RAJA_SIMDFORCE - macro to apply compiler-specific pragma that forces
```

Comment:
I see your point. I prefer the following policies: sequential (strictly sequential), simd (compiler vectorize if it thinks it should), and simd_force(force simd-ization -- code may actually run slower and code be incorrect if user doesn't know what she's doing). This gives a nice balance of prescriptive and descriptive options for users and is the most flexible. If you agree, I'll add the simd_force policy and unit tests.